### ISSUE TITLE
Linux compatible version of SDK

### ISSUE THREAD

**Issue by Heshmatkhah** [ZKTeco/Standalone-SDK#4]
Hi
I need to connect to your devices from a linux based application
Please release linux version

**Comment by WillSketcher254** [comment]
Yup, Ive been searching for a linux one too for months to avoid windows servers, did you get any lead on smth useful ?
